frontend/src/components/smart-contracts/hooks/useJob.ts

```diff
@@ -1,13 +1,20 @@
-import { useEffect, useState } from 'react';
 import { useSmartContracts } from 'components/smart-contracts/hooks/useSmartContracts';
 import { useWallet } from 'components/wallet/useWallet';
 import { BigNumber, ethers } from 'ethers';
 import {
   IJobData,
-  IJobSmartContractData,
   IJobMetaData,
+  IJobSmartContractData,
 } from 'interfaces/IJobData';
 import { ISmartContractState } from 'interfaces/ISmartContractState';
+import { useEffect, useState } from 'react';
+import { digestToCid, fetchIpfsMetaData } from 'services/ipfs';
+
+interface IJobCacheEntry {
+  job: IJobData;
+  v: number;
+  t: number;
```

Comment:
Can we name this something more descriptive?

Reply:
I added an enum to explain what these are.  I want the keys to be short since they are stored in localStorage.

Reply:
Oh! OK, makes sense.